### DISCUSSION TITLE
I didn't receive my Github Student Pack benefits even after 5 days of approval.

### DISCUSSION THREAD

**Discussion by Zeequeit** [community/community#138721] [GitHub Education] [answered]
### Hi everyone,

My Github Student pack is approved 5 days ago but still didn't receive benefits even after 5 days, what is the issue?

**Comment by davevad93** [comment, marked as answer]
Hi @Zeequeit ,

Don't worry it's perfectly normal, the waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks. 

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @Zeequeit was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
